content/web/tutorials/standalone-example.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Standalone Example-Web WorldWind/NASA WorldWind"
+date: 2017-11-15T14:33:47-08:00
+draft: false
+header: Standalone Example
+listdescription: Describes the how to host Web WorldWind on your own HTTP server.
+---
+
+## Hosting Locally
+
+---
+
+### The Data
+
+Get the [latest release](https://github.com/NASAWorldWind/WebWorldWind/releases/tag/v0.9.0-RC2) on GitHub.
+
+Host the WorldWind library and the images folder on your HTTP server:
+
+    https://YOUR_SERVER/worldwind.min.js
+    https://YOUR_SERVER/image/
+
+Get some data and unpack to:
```

Comment:
Clarification: the standalone folder is specific to the example data from NASA. With custom data, the folder could be named anything.